tests/functional/test_metamodel/test_multi_metamodel_refs.py

```diff
@@ -0,0 +1,116 @@
+import pytest  # noqa
+import os
+import os.path
+from pytest import raises
+from textx import metamodel_from_str, metamodel_from_file
+import textx.scoping.providers as scoping_providers
+import textx.scoping as scoping
+import textx.exceptions
+
+grammarA = """
+Model: a+=A;
+A:'A' name=ID;
+"""
+grammarB = """
+Model: b+=B;
+B:'B' name=ID '->' a=[A];
+"""
+grammarBWithImport = """
+Model: imports+=Import b+=B;
+B:'B' name=ID '->' a=[A];
+Import: 'import' importURI=STRING;
+"""
+
+
+def test_multi_metamodel_references1():
+    global_repo = scoping.GlobalModelRepository()
+    mm_A = metamodel_from_str(grammarA, global_repository=global_repo)
```

Comment:
@goto40  Does this means that in order to be *referencable* meta-model needs to be instantiated with a shared global repository? I wonder if one is going to reference meta-model from a third-party library how that is going to be achieved? 

I'm trying to understand the use case where one language developer is distributing his languages as a Python library and the another developer wants to reference elements contained in a model parsed by third-party meta-model/parser.

Reply:
An example is added to unittests, where three libs are simulated and the second lib uses the first and the third lib uses the second.

It also illustrates how to achieve a single global repo shared with an appropriate third-party lib.

However, the example illustrates, that a global_repo is not required in cases, where one model (including many others) is loaded... (comparison included in unittest). I also changed the docu (the global repo text was not 100% correct - and confusing).

Reply:
To make things clear:
 * Does this means that in order to be referencable meta-model needs to be instantiated with a shared global repository?
   * No.
 * I wonder if one is going to reference meta-model from a third-party library how that is going to be achieved (*to have a common repo*)?
   *  This only works, if the third party meta model already has a global repo. Then it can be achieved by passing the global_repo of the referencable meta-model to the new meta-model (like in the new unittest example).
   * But, as the unittest demonstrates, this is not required

Reply:
OK. That makes sense.